src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,29 @@
+import "./DropDown.scss";
+import topicArrow from "../../assets/upside-down-arrow-grey.svg";
+
+const DropDown = ({ topicTitle, lessonTitle, lessonType }) => {
+  return (
+    <nav className="drop-down">
+      <input type="checkbox" id="click" />
+      <label htmlFor="click">
+        <span className="drop-down__title">
+          {topicTitle}
+          <img src={topicArrow} alt="" className="drop-down__arrow" />
```

Comment:
For accessibility, we would want arrow as a button, so screen reader knows it's clickable